apps/auth-sample/build.gradle.kts

```
@@ -155,7 +154,8 @@ fun getValueFromEnvOrProperties(name: String): Any? {
 fun getValueFromProperties(name: String): String {
     val properties = gradleLocalProperties(rootDir)
     val property = properties[name] as? String
-    return property ?: throw GradleException("Missing property $name , please add it to the local.properties file")
+    return property
+        ?: throw GradleException("Missing property $name , please add it to the local.properties file")
```

Extreme nitpick (already was there) 😅 
```suggestion
        ?: throw GradleException("Missing property $name, please add it to the local.properties file")
```

---

Haha, no worries! 😁